CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,88 +8,62 @@
-        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
-
-        self.colorCountTextField.placeholder = "12"
+        collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
+        colorCountTextField.text = "\(Constants.defaultColorNumber)"
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-    }
-
-    // MARK: - Collection View Data Source Methods
-
-    func collectionView(_ collectionView:UICollectionView, numberOfItemsInSection section : Int) -> Int{
+    
+    // MARK: - CollectionViewDataSource
+    func collectionView(_ collectionView:UICollectionView, numberOfItemsInSection section : Int) -> Int {
         return ct!
     }
-
+    
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        var c = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! ColorsCollectionViewCell
-//        let color = UIColor.random
-//        c.backgroundColor = color
-//        c.hexColorLbl.text = color.description
-        c.configureCellWithColor(color: random_ui_color())
+        let c = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! ColorsCollectionViewCell
+        c.configureCell(withColor: random_ui_color())
         return c
     }
-
-
-
-
-    //////////////////////////////////
-    // MARK: - UITextFieldDelegate ///
-    //////////////////////////////////
-
+    
+    // MARK: - UITextFieldDelegate
     func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
-        if string.characters.count>0{
+        if string.characters.count > 0 {
             if (textField.text! as NSString).replacingCharacters(in: range, with: string).characters.count <= 2 {
-                var str = string.components(separatedBy: NSCharacterSet.decimalDigits.inverted).joined() // Filters string by removing non-decimal digits
+                let str = string.components(separatedBy: NSCharacterSet.decimalDigits.inverted).joined()
                 return string == str
-            } else {
-                return false
             }
+            return false
         }
-        else {
-            return true
-        }
+        return true
     }
-
-
-
-    // MARK: - IBActions
-
-    // This IBAction is called when the user taps on the background view
-    // It makes the colorsCountTextField resign first responder and sets colors count to the int value of the colorsCountTextField text
+    
+    // MARK: IBActions
     @IBAction func userDidTapOnTheBackgroundView(_ sender: UITapGestureRecognizer) {
         colorCountTextField.resignFirstResponder()
-        ct = Int(colorCountTextField.text ?? "")
+        ct = Int(colorCountTextField.text ?? "12")
```

Comment:
This is a good opportunity to use the constant you defined above instead of hardcoding 12 in.